accounts-db/src/tiered_storage/index.rs

```diff
@@ -47,10 +47,10 @@ pub struct IndexOffset(pub usize);
 )]
 pub enum IndexBlockFormat {
     /// This format optimizes the storage size by storing only account addresses
-    /// and offsets.  It skips storing the size of account data by storing account
-    /// block entries and index block entries in the same order.
+    /// and block offsets.  It skips storing the size of account data by storing
+    /// account block entries and index block entries in the same order.
     #[default]
-    AddressAndOffset = 0,
+    AddressAndBlockOffsetOnly = 0,
```

Comment:
Yep, I originally planned to have it in a separate PR, but since you mentioned that the comment about IndexWriterEntry isn't accurate, I think renaming this will be helpful as it brings out that the persisted format is actually different.

Will definitely in a separate PR in the future.  One PR one purpose :).